src/render/tutorials/adapt-service-without-webapi/1_5.html.md

```diff
@@ -0,0 +1,747 @@
+---
+title: Adapt Service without Web API
+description: "The Adapt Service without Web API Tutorial is the next step on our path to learning Knot.x. Today we will deal with data that comes directly from a database (we will not use a Web API layer this time). With just one simple Data Source Adapter, we will fetch the data and let Knot.x inject it into an HTML template."
+author: skejven
+layout: tutorial
+keywords: tutorial
+order: 5
+date: 2019-01-03
+knotxVersions:
+  - 1.5.0
+---
+## Overview
+
+Hello _Knot.x_ users!
+
+In this post we will show you how easy it is to inject data coming directly from a database into an HTML template.
+When developing advanced systems on the Web, we are often asked to integrate some external services and use
+the data our clients provide to render some information on a page. It is not a rare case when the
+data source we integrate with has no Web API or even can't have it because of security reasons.
+This is the case we will study over the course of this tutorial.
+
+What you're going to learn:
+- How to implement a simple Data Source Adapter for [Knot.x Data Bridge](https://github.com/Knotx/knotx-data-bridge)
+and start using it with _Knot.x_.
+- How to use [Vert.x JDBC client](http://vertx.io/docs/vertx-jdbc-client/java/) to easily access your database
+in a very performant way.
+
+## Solution Architecture
+
+So, we have a data source but no Web API to integrate with at the front-end layer.
+
+We have two options now:
+
+1. Implement a Web API layer to access the database and then integrate with it using e.g. AJAX or a HTTP Data Source Adapter. Please see [Getting Started with Knot.x Stack](http://knotx.io/tutorials/getting-started-with-knotx-stack/1_4/).
```

Comment:
We can refer to `1.5` version of the tutorial here.
`http://knotx.io/tutorials/getting-started-with-knotx-stack/1_5/`

Or maybe even better with no version?
Please update also other urls.